in-progress/gallery.yml

```diff
@@ -0,0 +1,141 @@
+# Installation notes:
+# - Set up OME RHEL 7 machine.
+# - Set up web server TLS in resources/certificates:
+#   star_openmicroscopy_org.crt+bundle, star_openmicroscopy_org.key
+# - Set host variables:
+#   database_password, server_password
+# - Run playbook.
+# - systemctl stop omero-{server,web}
+# - Migrate binary repository to /OMERO/ with rsync.
+# - Migrate database with pg_dump, pg_restore.
+# - Upgrade database.
+# - bin/omero db password for root user.
+# - systemctl start omero-{server,web}
+
+- hosts: omero-sls-gallery
+  roles:
+    - role: ome.network
+      network_ifaces:
```

Comment:
Can you put this internal configuration into a private host/group_vars file instead of the public playbook?

Reply:
Ah, I can try to follow what happens in `vlan-ome-idr-docker.yml` -- does something need to be written to make the playbook use it for that role though? Perhaps it's relevant that the host group gets the same name in the inventory.

Reply:
You just need to add it the the `vlan-ome-idr-docker` group and the vars will be automatically included

Reply:
Yay, will look at doing that in the other PR (after checking for conflicts).